docs/user/usage.rst

```diff
@@ -62,12 +67,14 @@ General syntax
    $ shaarli <global arguments> <endpoint> <endpoint arguments>
 
 
+.. note:: The following examples assume a :doc:`configuration` file is used
+
 GET info
 ~~~~~~~~
 
 .. code-block:: bash
 
-   $ shaarli -u https://host.tld/shaarli/ -s s3kr37 --output pprint get-info
+   $ shaarli --output pprint get-info
```

Comment:
the current formatting default is to output the plain HTTP response body to STDIN - https://github.com/shaarli/python-shaarli-client/blob/master/shaarli_client/main.py#L53

this flag is a bit ambiguous, it should become `--format`, with `--output` allowing to specify a filename to write data to; I'd rather wait for more input/output options to be available before adding these settings as configurable entries